web/packages/teleport/src/Roles/Roles.tsx

```diff
@@ -50,8 +50,9 @@ import { State, useRoles } from './useRoles';
 
 // RoleDiffProps are an optional set of props to render the role diff visualizer.
 type RoleDiffProps = {
-  RoleDiffComponent: ComponentType;
-  updateRoleDiff: (role: Role) => Promise<void>;
+  RoleDiffComponent: React.ReactElement;
```

Comment:
As this is now an element, normally the prop doesn't start with a capital letter

For example, `react-error-boundary`'s props

<img width="517" alt="image" src="https://github.com/user-attachments/assets/0d94c5d2-4603-498c-a178-e9de806c389e" />

(Also I think this should be a `ReactNode` - `react-error-boundary` and `React.PropsWithChildren` use `ReactNode`)